test/run-drun/inspect_message_blob.mo

```diff
@@ -0,0 +1,15 @@
+// NB: inspect_message cannot be tested in drun.
```

Comment:
then maybe `drun` should be enhanced?

Reply:
Maybe we, or we add a new class of test that uses dfx (though that inverts the dependency relationship). I'm not sure we have the resources to extend drun right now.